local-dev/setup_local_env.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -e
+
+# Helper script to set up Skaffold for local development.
+# This clones the terra-helm and terra-helmfile git repos, and templates in the desired
+#   Terra environment/k8s namespace to target.
+# If you need to pull changes to either terra-helm or terra-helmfile, rerun this script.
+
+# Required input
+ENV=$1
+# Optional input
+TERRA_HELM_BRANCH=${2:-master}
+TERRA_HELMFILE_BRANCH=${3:-master}
+
+# Clone Helm chart and helmfile repos
+rm -rf terra-helm
+rm -rf terra-helmfile
+git clone -b "$TERRA_HELM_BRANCH" --single-branch https://github.com/broadinstitute/terra-helm
+git clone -b "$TERRA_HELMFILE_BRANCH" --single-branch https://github.com/broadinstitute/terra-helmfile
+
+# Template in environment
+sed "s|ENV|${ENV}|g" skaffold.yaml.template > skaffold.yaml
+sed "s|ENV|${ENV}|g" values.yaml.template > values.yaml
```

Comment:
yuck. But I guess this is what we're doing for templating for now.
Maybe at least make the replaced string in the yaml more unique?

Reply:
sorry, not quite understand what you mean here. What we replace here is the env name, e.g., we have terra-yyu namespace in our k8s cluster. And what it does here by running `./setup_local_env.sh yyu`, it will generate the yaml which can be deployed to terra-yyu namespace. So the replaced string has to be "yyu", what you mean "make the replaced string in the yaml more unique"?

Reply:
I mean the `ENV` it's replacing in the .template files.
So here, `sed "s|MY_SPECIAL_ENV|${ENV}|g" skaffold.yaml.template > skaffold.yaml` and in the .template files replace `ENV` with `MY_SPECIAL_ENV`
 I don't know what a good replacement name is, since 'ENV' is very generic to start. idk, `DEV_ENV`, `DEV_NAMESPACE`, open to anything.

Reply:
"env" is what we used in terra-helm and helmfile:
https://github.com/broadinstitute/terra-helmfile/blob/master/helmfile.yaml#L10
Use env here because it does the corresponding terra env deployment. Use env here makes it clear what this does and how that works with helmfile?  What do you think? Do you want NAMESPACE instead?

Reply:
No,let's just go with ENV.